Cargo.toml

```diff
@@ -271,10 +271,15 @@ test = false
 doc = false
 
 [features]
+default = ["http-transport-curl"]
```

Comment:
I don't have any objection to this. Let me bring it to the Cargo team's meeting.